### ISSUE TITLE
Increase default passphrase to 6 words

### ISSUE THREAD

**Issue by atoponce** [mike-hearn/useapassphrase#3]
With a word list length of 5,837 unique words, entropy per word is about 12.51 bits. As such, the default 4 words only provide about 50 bits of entropy. This is on the weak side of things for fast offline password cracking.

Security could be improved either by increasing the word list significantly, which would impact performance, or increasing the passphrase length.

Best practice is at least 70 bits of entropy as a default to thwart offline password cracking. As such, your generator should be providing at least 6 words for the passphrase, which would provide about 75 bits of entropy.

**Comment by BrendanMitchellSisu** [comment]
Agreed, or add other options for increasing entropy (such as mixing in capital letters and digits). Granted, this removes some of the memorability advantage, but I do think these ought to be a tad stronger. The zxcvbn estimate isn't super applicable here.
